starport/cmd/scaffold.go

```diff
@@ -27,6 +27,7 @@ func NewScaffold() *cobra.Command {
 		Args:    cobra.ExactArgs(1),
 	}
 
+	c.AddCommand(NewScaffoldBandchain())
```

Comment:
I may think the commands are ordered in the same order they are defined here. I would say `band` should be below `packet`. Maybe @fadeev also has an opinion here

Reply:
Yes, `band` should be below `packet. Well spotted!